cmd/podman/main_local.go

```diff
@@ -153,3 +155,12 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 	}
 	return nil
 }
+func startTrace() {
+	if MainGlobalOpts.Trace {
+		span, _ = opentracing.StartSpanFromContext(Ctx, "stopCmd")
```

Comment:
That string isn't going to be accurate for most commands. Can we pull it out of the cobra.Command?